templates/default/070-components/UI-framework/MainControls/_ui-component_mode_info.scss

```diff
@@ -31,31 +38,37 @@ $il-mode-info-shadow: 0px 2px 2px 0px rgba(128, 128, 128, 1);
   }
 }
 
-
-.il-mode-info span.il-mode-info-content {
-  margin: auto;
+.il-mode-info-content,
+.il-mode-info-close {
   background-color: $il-mode-info-color;
-  padding: $il-padding-base-vertical $il-padding-base-horizontal $il-padding-base-vertical $il-padding-base-horizontal;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: $il-mode-info-height;
+  pointer-events: all;
+  position: relative;
+}
+
+.il-mode-info-content {
+  border-bottom-left-radius: calc($il-border-radius-large + 10px);
```

Comment:
Maybe use $il-border-radius-secondary-large instead of the calculation? We could consider making this variable 10 px as it doesn't seem to be currently in use anywhere else.